assets/resources/infrared/assets/tv.ir

```diff
@@ -1717,3 +1717,63 @@ type: raw
 frequency: 38000
 duty_cycle: 0.330000
 data: 3981 3992 495 1994 495 1995 496 1996 494 1996 495 1005 495 1006 494 1995 495 1997 494 1996 495 1996 494 1997 494 1996 495 1006 494 1005 495 1004 496 1005 495 1995 496 1994 496 1005 495 1004 496 1005 495 1006 494 1004 496 1006 494 8466 3978 3991 495 1994 495 1997 493 1994 496 1995 495 1004 496 1004 496 1996 494 1997 493 1996 494 1995 495 1995 495 1997 493 1004 495 1004 495 1006 494 1005 494 1998 491 1996 494 1006 494 1004 496 1006 494 1006 493 1005 495 1005 571
+# 
+name: Power
+type: parsed
+protocol: NEC
+address: 40 00 00 00
+command: 0B 00 00 00
+# 
+name: Vol_up
+type: parsed
+protocol: NEC
+address: 40 00 00 00
+command: 13 00 00 00
+# 
+name: Vol_dn
+type: parsed
+protocol: NEC
+address: 40 00 00 00
+command: 12 00 00 00
+# 
+name: Output
```

Comment:
Please carefully read https://github.com/flipperdevices/flipperzero-firmware/blob/dev/documentation/UniversalRemotes.md

Only `Power`, `Mute`, `Vol_up`, `Vol_dn`, `Ch_next`, and `Ch_prev` must be included